aerleon/lib/nftables.py

```diff
@@ -756,7 +767,9 @@ def _ProcessHeader(self, header_options):
             header_options.remove('noverbose')
         return netfilter_family, netfilter_hook, netfilter_priority, policy_default_action, verbose
 
-    def _ConfigurationDictionary(self, nft_pol):
+    def _ConfigurationDictionary(
+        self, nft_pol: List[Union[policy.Header, str, str, str, int, str, bool, DefaultDict]]
```

Comment:
This doesn't look right, this is a Union with duplicate entries:
```
Union[policy.Header, str, str, str, int, str, bool, DefaultDict]
```

Reply:
Ahh I should have added this one to the notes. I just changed it to be correct for now. This should actually be a tuple eventually as that would make more sense to pass in rather than a list of a bunch of different types.